connector/spanmetricsconnector/config.go

```diff
@@ -0,0 +1,80 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanmetricsconnector // import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/spanmetricsconnector"
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/featuregate"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+)
+
+const (
+	delta      = "AGGREGATION_TEMPORALITY_DELTA"
+	cumulative = "AGGREGATION_TEMPORALITY_CUMULATIVE"
+)
+
+var dropSanitizationGate = featuregate.GlobalRegistry().MustRegister(
+	"connector.spanmetrics.PermissiveLabelSanitization",
+	featuregate.StageAlpha,
+	featuregate.WithRegisterDescription("Controls whether to change labels starting with '_' to 'key_'"),
+)
```

Comment:
Will this be removed in a subsequent PR? Seems unnecessary to have a gate in a new component.

Reply:
I'm not quite sure, maybe we need to decide on these changes later. cc @kovrus .By the way, the lint for golangci-lint has always been timing out, it's strange.

Reply:
Yes, here is the issue https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18678

Reply:
Can we retry the lint job? It seems to have been timing out last night, but my local testing is normal.